### PR TITLE
Added documentation for v2.0 workflows

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
- [x] Create self-describing messages in the workflows
- [x] Add documentation to `CONTRIBUTING.md` describing the expected usage for each of the [v2.0](https://github.com/EthanThatOneKid/acmcsuf.com/milestone/1) workflows
  - [x] Add “Creating an Officer Update Request” section for `workflows/update_officer.yaml`
  - [x] Add section for `workflows/create_issue_channel.yaml`
  - [x] Add section for `workflows/close_issue_channel.yaml`

### CODE REVIEW

**Review by anhduy1202 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,6 +72,35 @@ Notice that the value `768px` is constant.
...
+## Custom Workflows
+
+In this repository, we use GitHub Actions to _automate all the things_ 😎
+
+### Creating an Officer Update Request
+
+To update the way an officer appears on <https://acmcsuf.com/about>, contributors may fill out our [**Officer Update Request Form**](https://github.com/EthanThatOneKid/acmcsuf.com/issues/new?assignees=&labels=automation%3Aofficer&template=officer_update_request.yaml&title=%5BOFFICER_AUTOMATION%5D) ([source](https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/.github/ISSUE_TEMPLATE/officer_update_request.yaml)) to proc a pull request to be generated with the requested changes.
+Contributors are free to merge the auto-generated pull request if it seems to have made the intended changes.
+
+The [`workflows/update_officer.yaml`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/.github/workflows/update_officer.yaml) workflow configuration will proc each time an issue is opened with the `automation:officer` label.
```
Comment (diamondburned) [inline CONTRIBUTING.md]: will "proc"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,6 +72,35 @@ Notice that the value `768px` is constant.
...
+This workflow relies on [`scripts/update-officer.js`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/main/scripts/update-officer.js) to make the appropriate changes to the checked-out codebase (including updating the repo's `officers.json` file or adding/replacing their image in the repo's `/static/assets/authors/` directory).
+Then, the workflow generates a pull request based on the changes made by the `update-officer.js` script and immediately closes the issue.
+
+### Automated Third-Party Message Board
+
+Our team uses Discord as our third-party messaging service of choice to discuss relevant topics that aren't suited/ready for GitHub.
+We have text channels set up on Discord dedicated to every open issue on GitHub.
+This approach allowed us to communicate about multiple issues in an orderly and stress-free fashion.
```
Comment (diamondburned) [inline CONTRIBUTING.md]: Nitpick: s/allowed us/allows us

File: CONTRIBUTING.md (hunk 1 above)
Comment (EthanThatOneKid) [reply CONTRIBUTING.md]: Replaced word with "automate".

File: CONTRIBUTING.md (hunk 2 above)
Comment (EthanThatOneKid) [reply CONTRIBUTING.md]: Thanks for catching that. Got it.

**Review by diamondburned — APPROVED** [review]

**Pull request merged**